experiments/tester/src/RNTester/TestComponents/ThemeTest.tsx

```diff
@@ -75,17 +66,43 @@ const getThemedStyles = themedStyleSheet((theme: ITheme) => ({
     fontWeight: theme.typography.weights.medium,
     fontFamily: theme.typography.families.primary,
     marginBottom: 5
-  } as ReactNative.TextStyle
+  } as ReactNative.TextStyle,
+  stackStyle: {
+    borderWidth: 2,
+    borderColor: theme.colors.focusBorder,
+    padding: 12,
+    margin: 8,
+    backgroundColor: theme.colors.background
+  }
 }));
 
 const styles = ReactNative.StyleSheet.create({
   swatchItem: {
     flexDirection: 'row',
     alignItems: 'center',
     marginVertical: 5
+  },
+  pickerContainer: {
+    flexDirection: 'row',
+    justifyContent: 'space-evenly'
   }
 });
 
+const Panel: React.FunctionComponent = () => {
+  const [disabled, setDisabled] = React.useState(false);
+  const onClick = React.useCallback(() => setDisabled(!disabled), [disabled, setDisabled]);
```

Comment:
I don't think `onClick` is worth caching if it depends on `disabled`. In all the usage below, if `disabled` changes, it will cause the buttons to re-render anyways.

Reply:
It can still avoid memory churn if the Panel re-renders for another reason, though, right?

i.e. if useTheme() changes, the stuff will rerender, but the onClick lambda expression won't necessarily be recreated